Sage/carbon
# fix(flat-table-row): allow click actions inside interactive row

the ::after pseudo-element used for the focus state of an interactive FlatTableRow is positioned on
top of the row and blocks any mouse events from taking place inside its individual cells. Adding
`pointer-events: none` to this pseudo-element fixes the problem.

fix #5402

### Proposed behaviour

Links and other interactive content should work when clicked inside of a focused expandable row.



### Current behaviour

Links etc. do not work on click when inside an expandable row which is currently focused



### Checklist




#### QA


### Additional context



### Testing instructions



Codesandbox demo here (forked from the one in the original Github issue, to use supported versions of React and styled-components): https://codesandbox.io/s/kind-leftpad-tuo2ly?file=/src/index.js

The following CodeSandbox is an example of the broken behaviour.
You can see the new behaviour by looking at the version in the comment by `codesandbox[bot]`.
